odl/test/solvers/nonsmooth/douglas_rachford_test.py

```diff
@@ -92,7 +92,7 @@ def test_primal_dual_l1():
     x = space.zero()
     douglas_rachford_pd(x, f, g, L, tau=3.0, sigma=[1.0], niter=10)
 
-    assert all_almost_equal(x, data_1, places=2)
+    assert all_almost_equal(x, data_1, ndigits=2)
```

Comment:
Why do we have `HIGH_ACCURACY` and  `LOW_ACCURACY` defined above, and anyway hard-code `ndigits` here?

Reply:
Dunno, I just took the code as-is. Probably copy & paste.

Reply:
I think here the number of expected correct digits depends on the problem and on the (few) iterations we run. I'll keep it like this.